01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,14 @@
+numbers = 0
+
+20.times do
+	numbers += 1
+	if numbers % 3 == 0 && numbers % 5 == 0
+    puts "FizzBuzz"
+	elsif numbers % 3 == 0
+    puts "Fizz"
+	elsif numbers % 5 == 0
+		puts "Buzz"
+	else
+		puts numbers
+	end
+end
```

Comment:
インデントに「タブ」や「スペース」が混じっているので統一しましょう。
Rubyでは「タブ」は使わず。スペース２つでインデントを整えます。

```suggestion
20.times do
  numbers += 1
  if numbers % 3 == 0 && numbers % 5 == 0
    puts 'FizzBuzz'
  elsif numbers % 3 == 0
    puts 'Fizz'
  elsif numbers % 5 == 0
    puts 'Buzz'
  else
    puts numbers
  end
end
```

Reply:
すみません。
スペース２つでインデントとするのは存じておりました。
しかし、Vimの設定でタブ幅がスペース２つ分となっていて、かつタブとスペースが視覚的に区別しにくくなっていたため確認漏れしてしまいました。
先程、タブ幅スペース４つ分にし、タブには印がつくようにVimを設定しなおしました。
また、これからはGitHub.com上でも確認します。

Reply:
> しかし、Vimの設定でタブ幅がスペース２つ分となっていて、かつタブとスペースが視覚的に区別しにくくなっていたため確認漏れしてしまいました。

なるほど、了解です！
修正いただいて感謝です！